lib/agent/author_name.rb

```diff
@@ -50,10 +50,6 @@ def full_name
     # an 'or' conjuction is likely to generate results that mostly match this variant,
     # but additional variants might add something when using an 'ExactMatch' search.
     # @return [String] name(s) to be queried in an OR (disjunction) query
-    def text_search_query
```

Comment:
Do you need to get rid of the comments too?

Reply:
I believe that comment also makes sense given what `text_search_query` returns